src/Components/Scenes/SideNavigation/RequestListItem/RequestListItem.js

```diff
@@ -25,23 +25,36 @@ const requestListItem = makeStyles(theme => ({
   }
 }));
 
-const RequestListItem = ({ closeAllArrows, requestListItemsName }) => {
+const RequestListItem = ({
+  closeAllArrows,
+  requestListItemsName,
+  requestInboxAlert,
+  newAlerts
+}) => {
   const classes = requestListItem();
   const [openIconThree, setOpenIconThree] = useState(false);
+  const [newAlert, setAlert] = useState([]);
 
   useEffect(() => {
     if (closeAllArrows === false && openIconThree === true) {
       setOpenIconThree(false);
     }
   }, [closeAllArrows, openIconThree]);
 
+  useEffect(() => {
+    requestInboxAlert();
+    setAlert(newAlerts);
+  }, [requestInboxAlert, newAlerts]);
+
   const requestListItemText = requestListItemsName.map(text => (
     <ListItem key={text} button className="Nav__nested-item">
       <ListItemText className="Nav__text">
         {text === 'Inbox' ? (
           <>
             {text}
-            <span className={classes.newInbox}> (3 new)</span>
+            {newAlert > 0 ? (
```

Comment:
Here and on line 79 - did you mean to check `newAlert.length` ? Because your `useState()` call starts with an array, I'm assuming that variable is a list of alerts yeah?

Also may make sense to rename it to `[alerts, setAlerts]` then